hassio/src/hassio-panel-router.ts

```diff
@@ -26,6 +27,8 @@ class HassioPanelRouter extends HassRouterPage {
 
   @property({ attribute: false }) public supervisorInfo: HassioSupervisorInfo;
 
+  @property({ attribute: false }) public hassioInfo!: HassioInfo;
```

Comment:
`{ attribute: false }` is the default: https://github.com/Polymer/lit-element/blob/master/src/lib/updating-element.ts#L438-L446

Reply:
It is because that freaking lit plugin complains about it...

Reply:
Can we disable that rule and file a bug?

Reply:
Reading the code you posted ^ `{ attribute: false }` would not add it to `observedAttributes` whereas leaving it out would add it to `observedAttributes` as `hassioinfo` with type `string`.

So it is not the same, now we would never set `hassioinfo` so no problem for us.

But it is not a bug from `ts-lit-plugin` as there would be no way to convert the value set to the attribute `hassioinfo` to be of the type `HassioInfo` as the default for an attribute is a `string`.

Reply:
Oh, interesting.